doc/build-helpers/testers.chapter.md

```diff
@@ -154,12 +154,20 @@ testers.shellcheck {
 
 ### Inputs {#tester-shellcheck-inputs}
 
-[`src` (path or string)]{#tester-shellcheck-param-src}
+`name` (string, optional)
+: The name of the test, optionally prefixed with `shellcheck-` if `useNamePrefix` is `true`.
```

Comment:
What is the purpose of the `useNamePrefix` switch? I'd say that a good `name` should always include the `shellcheck-` prefix.

Reply:
Removed in latest -- good feedback!

Reply:
> Removed in latest -- good feedback!

I don't see it is done. Maybe you committed locally and forgot to push?

Reply:
I did :(

Pushed it this time :)